pkgs/development/libraries/science/math/or-tools/default.nix

```diff
@@ -18,53 +18,57 @@
 , zlib
 }:
 
+let
+  pybind11_protobuf = fetchFromGitHub {
+    owner = "pybind";
+    repo = "pybind11_protobuf";
+    rev = "b713501f1da56d9b76c42f89efd00b97c26c9eac";
```

Comment:
According to https://github.com/google/or-tools/blob/v9.7/WORKSPACE#L228, this should be `5baa2dc9d93e3b608cde86dfa4b8c63aeab4ac78`.

Reply:
I had to use `b713501f1da56d9b76c42f89efd00b97c26c9eac` for the build to succeed, probably due to some mismatches as in google/or-tools#4027.